src/components/FileHistory/index.jsx

```diff
@@ -15,7 +16,7 @@ const trackEvent = () => {
 const FileHistory = ({ params: { fileId } }) => {
   trackEvent()
   return (
-    <Query query={client => client.get('io.cozy.files', fileId)}>
+    <Query query={() => Q('io.cozy.files').getById(fileId)}>
```

Comment:
Not part of this PR : both queries could be named and have a fetch policy.

Reply:
https://github.com/cozy/cozy-drive/issues/2248